### PR TITLE
Update DESCRIPTION

### PR DESCRIPTION
**Pull request by salim-b**
- add ORCID for @salim-b 
- sort `Suggests` alphabetically

### CODE REVIEW

File: DESCRIPTION [hunk 1]
```diff
@@ -1,9 +1,9 @@
 Package: extraInserts
 Title: RStudio Addin to Insert Various (Pipe) Operators
 Version: 0.0.0.9000
-Authors@R: as.person(c(
-    "Konrad Zdeb <konrad.zdeb@me.com> [aut, cre]", 
-    "Salim Brüggemann <salim-b@pm.me> [aut]"
+Authors@R: c(
+    person("Konrad", "Zdeb", role = c("aut", "cre"), email = "konrad.zdeb@me.com"),
+    person("Salim", "Brüggemann", role = "aut", email = "salim-b@pm.me", comment = c(ORCID = "0000-0002-5329-5987")),
   ))
```
Comment (konradzdeb) [inline DESCRIPTION]: Build failed as the `DESCRIPTION` file appears to be malformed. I think it's just a minor error with a superfluous bracket at the end...
```suggestion
    person("Salim", "Brüggemann", role = "aut", email = "salim-b@pm.me", comment = c(ORCID = "0000-0002-5329-5987")))
```

**Review by konradzdeb — CHANGES_REQUESTED** [review]
Hi, 
Thanks for the suggestion, happy to merge if you can fix the issue so the build passes.

**Comment by salim-b** [comment]
> Thanks for the suggestion, happy to merge if you can fix the issue so the build passes.

It's fixed now. I'm sorry for the broken build, I made the changes without testing 🙈

**Comment by konradzdeb** [comment]
Hi, no probs. Thanks for the fix.

**Pull request merged**